dspace_metadata_import.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+
+########################################################################################
+## Run this script to import metadata from old dspace into new dspace6 database, and  ##
+## copy over the metadata registry files.											  ##
+########################################################################################
+
+
+## Refer to : https://unix.stackexchange.com/questions/191990/shell-script-to-execute-psql-command ##
+## DSpace6 Upgrading: https://wiki.duraspace.org/display/DSDOC6x/Upgrading+DSpace ##
+
+
+#########################################################################
+#                       									 			#
+#	Need to 															#
+#		1) copy over the dumped pgsql file from old dspace, and			#
+#		2) set up the ownership, and									#
+#		3) replace the user to be dspace or something, and 				#
+#		4) comment out the block of adding rule protect_approval_upd	#
+#    	5) comment out the privileges granted to user libacct			#
+#                       												#
+#########################################################################
+
+PATH_TO_REGISTRIES_FROM_OLD_DSPACE = /vagrant/registries.tar.gz
+PATH_TO_OLD_DSPACE_DATABASE_DATA = /vagrant/test3.pgsql
+
+
+echo "#####  Create new database dspace2  #####"
+
+sudo -u postgres psql <<EOF
+
+DROP DATABASE IF EXISTS dspace2;
```

Comment:
We'll probably need to make the database name a variable here, since it'll be different in different contexts.